### ISSUE TITLE
Saving the predicted labelled image to 8-bit instead of 16-bit, and using the right number of labels

### ISSUE THREAD

**Issue by gtrancourt** [masonearles/RandomForestMicroCT#2]
Currently, the predicted image, `RFPredictCTStack_out`, is saved to 16-bits using `img_as_int`. This is too much since there as so few labels, so `img_as_ubyte` (saving to 8-bits) should be used. 

Further, since the number of labels could change from one user to another, and since `img_as_` requires values between 0 and 1 (i.e. they must be divided by the total number of labels), there should be a call to the actual number of labels. I suggest the following, but the `np.unique` could actually be called within the `img_as_ubyte` function:

```python
uniq_labs = np.unique(RFPredictCTStack_out[1])
io.imsave(filepath + 'FILENAME.tif', img_as_ubyte(RFPredictCTStack_out / len(uniq_labs)))
```

**Comment by mattjenkins3** [comment]
Good idea to use img_as_ubyte, rather than img_as_int. The files are smaller and I cannot fathom a situation in which a user will need to segment >255 different classes.

Also, my apologies for not having the updated script on GitHub yet, but I already built in a method for checking the number of classes before saving the predicted stack. I used the same method, actually.
```python
hold = len(np.unique(RFPredictCTStack_out[1]))
```
Look for the changes on branch 'add_changes' later today.
